L/Lingeling/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Lingeling"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/arminbiere/lingeling.git", "89a167d0d2efe98d983c87b5b84175b40ea55842"),
+    DirectorySource("./bundled")
+    ]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/lingeling*
+cp ../makefile .
+sed -i '/#include *<stdlib.h>/a #include <stdint.h>' *.c
+sed -i '/#include *<stdlib.h>/a #include <stdint.h>' *.h
```

Comment:
Can you please submit a PR usptream to fix this?

Reply:
There already is a (very old) PR for this, but I am not positive it will ever get fixed:
https://github.com/arminbiere/lingeling/pull/4